This file is part of MXE.
See index.html for further information.

From 31ed00623baaa2f62ed3b0749f9acecc94ece85d Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sat, 25 Jul 2015 17:15:43 +0300
Subject: [PATCH] disable samples/inline

This sample requires running pire_inline, which is impossible
in case of cross-compilation.
---
 configure.ac        | 1 -
 samples/Makefile.am | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index e185cb9..eafe6bb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -101,7 +101,6 @@ AC_CONFIG_FILES([
 	tools/Makefile
 	tools/bench/Makefile
 	samples/Makefile
-	samples/inline/Makefile
 	samples/blacklist/Makefile
 	samples/pigrep/Makefile
 ])
diff --git a/samples/Makefile.am b/samples/Makefile.am
index 32adaf3..1acc17e 100644
--- a/samples/Makefile.am
+++ b/samples/Makefile.am
@@ -1,4 +1,3 @@
 SUBDIRS = \
-	inline \
 	blacklist \
 	pigrep
-- 
1.9.1

